Header header #Time and sequence number matches PlanningRequest.msg
# Output path. Each point has
# Position to reach, vector to point the camera, speed etc. as defined below
Waypoint[] plan

uint16 vehicle_id
